nautobot/core/testing/filters.py

```diff
@@ -145,10 +145,13 @@ def test_name(self):
 
         def test_slug(self):
             """Verify that the filterset supports filtering by slug."""
-            params = {"slug": self.queryset.values_list("slug", flat=True)[:2]}
-            filterset = self.filterset(params, self.queryset)
-            self.assertTrue(filterset.is_valid())
-            self.assertEqual(filterset.qs.count(), 2)
+            # slug field has been removed from alot of models,
+            # causing this test to fail for such models
```

Comment:
Slugless models shouldn't be using `NameSlugFilterTestCase` :-) -- maybe we need a new `NameOnlyFilterTestCase` or maybe `NameCommentsFilterTestCase` or even `NameCommentsDescriptionFilterTestCase`?

Reply:
Illd go with `NameOnlyFilterTestCase` as not all models has both or either comments and description